src/python_testing/TC_RVCCLEANM_3_2.py

```diff
@@ -92,17 +113,17 @@ class CommonCodes(Enum):
                     new_mode = m.mode
                     break
 
-            self.print_step(5, "Send ChangeToMode command with NewMode set to %d" % (new_mode))
+            self.print_step(7, "Send ChangeToMode command with NewMode set to %d" % (new_mode))
 
             ret = await self.send_change_to_mode_cmd(newMode=new_mode)
             asserts.assert_true(ret.status == CommonCodes.SUCCESS.value, "Changing the mode should succeed")
 
         self.default_controller.ExpireSessions(self.dut_node_id)
 
-        self.print_step(6, "Physically power cycle the device")
+        self.print_step(8, "Physically power cycle the device")
         input("Press Enter when done.\n")
 
-        self.print_step(7, "Read CurrentMode attribute")
+        self.print_step(9, "Read CurrentMode attribute")
```

Comment:
This may need a small addition. See here: https://github.com/CHIP-Specifications/chip-test-plans/pull/3166/files#r1275333253